lib/daru/vector.rb

```diff
@@ -879,41 +879,67 @@ def to_s
       to_html
     end
 
-    # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
+    # Create a summary of the Vector
+    # @params [Fixnum] indent_level
+    # @return [String] String containing the summary of the Vector
+    # @example
+    #   dv = Daru::Vector.new [1, 2, 3]
+    #   puts dv.summary
+    #
+    #   =
+    #     n :3
+    #     non-missing:3
+    #     median: 2
+    #     mean: 2.0000
+    #     std.dev.: 1.0000
+    #     std.err.: 0.5774
+    #     skew: 0.0000
+    #     kurtosis: -2.3333
```

Comment:
Make an example output docs this way:
```ruby
# @example
#   dv = Daru::Vector.new [1, 2, 3]
#   puts dv.summary
#
#   # =
#   #   n :3
#   #   non-missing:3
#   #   median: 2
#   #   mean: 2.0000
#   #   std.dev.: 1.0000
#   #   std.err.: 0.5774
#   #   skew: 0.0000
#   #   kurtosis: -2.3333
```
Everything under the `@example` will be highlighted as a Ruby code in docs, and output will be highlighted pretty strangely, so we should make it look as a "comments".